gatsby-config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   siteMetadata: {
-    title: `Gatsby Default Starter`,
-    description: `Kick off your next, great Gatsby project with this default starter. This barebones starter ships with the main Gatsby configuration files you might need.`,
+    title: `Zenika Open Source`,
+    description: `Insights about Zenika's Open Source contributions`,
     author: `@gatsbyjs`,
```

Comment:
Is there a twitter for Zenika Open Source ? If it does not exisit we can put @ZenikaIT as author.

Reply:
I don't think we do. Could you confirm it @frinyvonnick? If so, I will set it to @ZenikaIT as you mentioned 😄

Reply:
The twitter account @ZenikaOSS has been created :ok_hand: